### ISSUE TITLE
story:build fails with Nuxt plugin

### ISSUE THREAD

**Issue by smares** [histoire-dev/histoire#492]
### Describe the bug

Using the following `histoire.config.ts`:

```ts
import { defineConfig } from 'histoire'
import { HstVue } from '@histoire/plugin-vue'
import { HstNuxt } from '@histoire/plugin-nuxt'

export default defineConfig({
  plugins: [
    HstVue(),
    HstNuxt(),
  ],
})
```

Running `yarn histoire:build` fails with the following error:

```
smares@PC:~/my-app$ yarn story:build
ℹ Using Tailwind CSS from ~/assets/css/tailwind.css                                                                                                                                                                                  nuxt:tailwindcss 10:15:45

 WARN  Sourcemap for "/home/smares/my-app/node_modules/@nuxtjs/i18n/dist/runtime/plugins/composition.mjs" points to missing source files                                                                                              10:15:46


 WARN  Sourcemap for "/home/smares/my-app/node_modules/@nuxtjs/i18n/dist/runtime/plugins/i18n.mjs" points to missing source files                                                                                                     10:15:46

ℹ Vite client warmed up in 1366ms                                                                                                                                                                                                                     10:15:47
✔ Nitro built in 860 ms                                                                                                                                                                                                                         nitro 10:15:48
Using 19 threads for story collection
vite config hook [Function (anonymous)]
vite v4.1.4 building for development...
✓ 38 modules transformed.
Unexpected token (Note that you need plugins to import files that are not JavaScript)
file: /__resolved__virtual:$histoire-config:1:2897
1: export const config = {"plugins":[{"name":"builtin:tailwind-tokens"},{"name":"builtin:vanilla-support","supportPlugin":{"id":"vanilla","moduleName":"/home/smares/my-app/node_modules/histoire/dist/node/builtin-plugins/vanilla-support","setupFn":"setupVanilla"}},{"name":"@histoire/plugin-vue","supportPlugin":{"id":"vue3","moduleName":"@histoire/plugin-vue","setupFn":"setupVue3","importStoriesPrepend":"import { defineAsyncComponent as defineAsyncComponentVue3 } from 'vue'"},"commands":[{"id":"histoire:plugin-vue:generate-story","label":"Generate Vue 3 story from component","icon":"https://vuejs.org/logo.svg","searchText":"generate create","clientSetupFile":"@histoire/plugin-vue/dist/commands/generate-story.client.js"}]},{"name":"@histoire/plugin-nuxt"}],"outDir":"/home/smares/my-app/.histoire/dist","storyMatch":["**/*.story.vue","**/*.story.svelte"],"storyIgnored":["**/node_modules/**","**/dist/**"],"supportMatch":[{"id":"vanilla","patterns":["**/*.js"],"pluginIds":["vanilla"]},{"id":"vue","patterns":["**/*.vue"],"pluginIds":["vue3"]}],"tree":{"file":"title","order":"asc"},"theme":{"title":"Histoire","colors":{"primary":{"50":"#ecfdf5","100":"#d1fae5","200":"#a7f3d0","300":"#6ee7b7","400":"#34d399","500":"#10b981","600":"#059669","700":"#047857","800":"#065f46","900":"#064e3b"},"gray":{"50":"#fafafa","100":"#f4f4f5","200":"#e4e4e7","300":"#d4d4d8","400":"#a1a1aa","500":"#71717a","600":"#52525b","700":"#3f3f46","750":"#323238","800":"#27272a","850":"#1f1f21","900":"#18181b","950":"#101012"}},"defaultColorScheme":"auto","storeColorScheme":true,"darkClass":"dark"},"responsivePresets":[{"label":"Mobile (Small)","width":320,"height":560},{"label":"Mobile (Medium)","width":360,"height":640},{"label":"Mobile (Large)","width":414,"height":896},{"label":"Tablet","width":768,"height":1024},{"label":"Laptop (Small)","width":1024,"height":null},{"label":"Laptop (Large)","width":1366,"height":null},{"label":"Desktop","width":1920,"height":null},{"label":"4K","width":3840,"height":null}],"backgroundPresets":[{"label":"Transparent","color":"transparent","contrastColor":"#333"},{"label":"White","color":"#fff","contrastColor":"#333"},{"label":"Light gray","color":"#aaa","contrastColor":"#000"},{"label":"Dark gray","color":"#333","contrastColor":"#fff"},{"label":"Black","color":"#000","contrastColor":"#eee"}],"sandboxDarkClass":"dark","routerMode":"history","build":{"excludeFromVendorsChunk":[{}]},"vite":{"server":{"watch":{},"fs":{"allow":["/home/smares/my-app/node_modules/nuxt/dist/app","/home/smares/my-app","/home/smares/my-app/.nuxt","/home/smares/my-app","/home/smares/my-app","/home/smares/my-app","/home/smares/my-app/node_modules","/home/smares/my-app/node_modules"]},"middlewareMode":false},"define":{"__NUXT_VERS...
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ^
2: export const logos = {}
file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:2105
        base = Object.assign(new Error(base.message), base);
                             ^

Error [RollupError]: Unexpected token (Note that you need plugins to import files that are not JavaScript)
    at error (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:2105:30)
    at Module.error (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:13174:16)
    at Module.tryParse (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:13851:25)
    at Module.setSource (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:13461:39)
    at ModuleLoader.addModuleSource (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:23422:20) {
  cause: SyntaxError: Unexpected token (1:2897)
      at pp$4.raise (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20830:13)
      at pp$9.unexpected (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:18131:8)
      at pp$5.parsePropertyValue (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20551:17)
      at pp$5.parseProperty (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20500:8)
      at pp$5.parseObj (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20455:21)
      at pp$5.parseExprAtom (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20190:17)
      at pp$5.parseExprSubscripts (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20006:19)
      at pp$5.parseMaybeUnary (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:19972:17)
      at pp$5.parseExprOps (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:19899:19)
      at pp$5.parseMaybeConditional (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:19882:19)
      at pp$5.parseMaybeAssign (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:19849:19)
      at pp$5.parsePropertyValue (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20509:87)
      at pp$5.parseProperty (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20500:8)
      at pp$5.parseObj (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20455:21)
      at pp$5.parseExprAtom (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20190:17)
      at pp$5.parseExprSubscripts (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20006:19)
      at pp$5.parseMaybeUnary (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:19972:17)
      at pp$5.parseExprOps (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:19899:19)
      at pp$5.parseMaybeConditional (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:19882:19)
      at pp$5.parseMaybeAssign (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:19849:19)
      at pp$5.parsePropertyValue (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20509:87)
      at pp$5.parseProperty (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20500:8)
      at pp$5.parseObj (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20455:21)
      at pp$5.parseExprAtom (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20190:17)
      at pp$5.parseExprSubscripts (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:20006:19)
      at pp$5.parseMaybeUnary (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:19972:17)
      at pp$5.parseExprOps (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:19899:19)
      at pp$5.parseMaybeConditional (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:19882:19)
      at pp$5.parseMaybeAssign (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:19849:19)
      at pp$8.parseVar (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:18674:24)
      at pp$8.parseVarStatement (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:18538:8)
      at pp$8.parseStatement (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:18286:17)
      at Parser.parseExport (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:24690:33)
      at pp$8.parseStatement (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:18307:74)
      at pp$8.parseTopLevel (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:18188:21)
      at Parser.parse (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:17960:15)
      at Function.parse (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:18010:35)
      at Graph.contextParse (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:24406:38)
      at Module.tryParse (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:13848:31)
      at Module.setSource (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:13461:39)
      at ModuleLoader.addModuleSource (file:///home/smares/my-app/node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:23422:20) {
    pos: 2897,
    loc: Position { line: 1, column: 2897 },
    raisedAt: 2900
  },
  code: 'PARSE_ERROR',
  id: '/__resolved__virtual:$histoire-config',
  pos: 2897,
  loc: {
    column: 2897,
    file: '/__resolved__virtual:$histoire-config',
    line: 1
  },
  frame: `1: export const config = {"plugins":[{"name":"builtin:tailwind-tokens"},{"name":"builtin:vanilla-support","supportPlugin":{"id":"vanilla","moduleName":"/home/smares/my-app/node_modules/histoire/dist/node/builtin-plugins/vanilla-support","setupFn":"setupVanilla"}},{"name":"@histoire/plugin-vue","supportPlugin":{"id":"vue3","moduleName":"@histoire/plugin-vue","setupFn":"setupVue3","importStoriesPrepend":"import { defineAsyncComponent as defineAsyncComponentVue3 } from 'vue'"},"commands":[{"id":"histoire:plugin-vue:generate-story","label":"Generate Vue 3 story from component","icon":"https://vuejs.org/logo.svg","searchText":"generate create","clientSetupFile":"@histoire/plugin-vue/dist/commands/generate-story.client.js"}]},{"name":"@histoire/plugin-nuxt"}],"outDir":"/home/smares/my-app/.histoire/dist","storyMatch":["**/*.story.vue","**/*.story.svelte"],"storyIgnored":["**/node_modules/**","**/dist/**"],"supportMatch":[{"id":"vanilla","patterns":["**/*.js"],"pluginIds":["vanilla"]},{"id":"vue","patterns":["**/*.vue"],"pluginIds":["vue3"]}],"tree":{"file":"title","order":"asc"},"theme":{"title":"Histoire","colors":{"primary":{"50":"#ecfdf5","100":"#d1fae5","200":"#a7f3d0","300":"#6ee7b7","400":"#34d399","500":"#10b981","600":"#059669","700":"#047857","800":"#065f46","900":"#064e3b"},"gray":{"50":"#fafafa","100":"#f4f4f5","200":"#e4e4e7","300":"#d4d4d8","400":"#a1a1aa","500":"#71717a","600":"#52525b","700":"#3f3f46","750":"#323238","800":"#27272a","850":"#1f1f21","900":"#18181b","950":"#101012"}},"defaultColorScheme":"auto","storeColorScheme":true,"darkClass":"dark"},"responsivePresets":[{"label":"Mobile (Small)","width":320,"height":560},{"label":"Mobile (Medium)","width":360,"height":640},{"label":"Mobile (Large)","width":414,"height":896},{"label":"Tablet","width":768,"height":1024},{"label":"Laptop (Small)","width":1024,"height":null},{"label":"Laptop (Large)","width":1366,"height":null},{"label":"Desktop","width":1920,"height":null},{"label":"4K","width":3840,"height":null}],"backgroundPresets":[{"label":"Transparent","color":"transparent","contrastColor":"#333"},{"label":"White","color":"#fff","contrastColor":"#333"},{"label":"Light gray","color":"#aaa","contrastColor":"#000"},{"label":"Dark gray","color":"#333","contrastColor":"#fff"},{"label":"Black","color":"#000","contrastColor":"#eee"}],"sandboxDarkClass":"dark","routerMode":"history","build":{"excludeFromVendorsChunk":[{}]},"vite":{"server":{"watch":{},"fs":{"allow":["/home/smares/my-app/node_modules/nuxt/dist/app","/home/smares/my-app","/home/smares/my-app/.nuxt","/home/smares/my-app","/home/smares/my-app","/home/smares/my-app","/home/smares/my-app/node_modules","/home/smares/my-app/node_modules"]},"middlewareMode":false},"define":{"__NUXT_VERS...\n` +
    '                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ^\n' +
    '2: export const logos = {}',
  watchFiles: [
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundle-main.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundle-sandbox.js',
    '/home/smares/my-app/node_modules/@histoire/app/package.json',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/index.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/style.css',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/style.css',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/sandbox.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/app/style/sandbox.css',
    '/home/smares/my-app/node_modules/@histoire/vendors/package.json',
    '/home/smares/my-app/node_modules/@histoire/vendors/dist/client/b-vue.js',
    '/home/smares/my-app/node_modules/@histoire/vendors/dist/client/b-pinia.js',
    '/home/smares/my-app/node_modules/@histoire/vendors/dist/client/b-floating-vue.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/App.vue.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/router.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/plugin.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/App.vue2.js',
    '/__resolved__virtual:$histoire-theme.css',
    '/home/smares/my-app/node_modules/@histoire/shared/package.json',
    '/home/smares/my-app/node_modules/@histoire/vendors/dist/client/b-vue-router.js',
    '/home/smares/my-app/node_modules/@histoire/shared/dist/index.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/story/GenericMountStory.vue.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/story/GenericRenderStory.vue.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/util/mapping.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/util/const.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/util/preview-settings.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/util/dark.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/util/config.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/util/state.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/story/GenericMountStory.vue2.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/story/GenericRenderStory.vue2.js',
    '/__resolved__virtual:$histoire-stories',
    '/home/smares/my-app/node_modules/vue/package.json',
    '/home/smares/my-app/node_modules/vue/dist/vue.runtime.esm-bundler.js',
    '/home/smares/my-app/node_modules/@histoire/vendors/dist/client/vue.runtime.esm-bundler.js',
    '/home/smares/my-app/node_modules/@histoire/vendors/dist/client/index.js',
    '/home/smares/my-app/node_modules/@histoire/vendors/dist/client/index2.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/HomeView.vue.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/story/StoryView.vue.js',
    '\x00vite/preload-helper',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/tree/StoryList.vue.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/base/BaseSplitPane.vue.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/app/AppHeader.vue.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/stores/story.js',
    '/home/smares/my-app/node_modules/@histoire/vendors/dist/client/b-vue-use.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/util/keyboard.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/util/responsive.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/stores/command.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/app/Breadcrumb.vue.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/search/SearchModal.vue.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/command/CommandPromptsModal.vue.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/app/InitialLoading.vue.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/app/AppHeader.vue2.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/tree/StoryList.vue2.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/search/SearchModal.vue2.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/command/CommandPromptsModal.vue2.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/app/InitialLoading.vue2.js',
    '/home/smares/my-app/node_modules/@histoire/shared/dist/codegen/index.js',
    '/home/smares/my-app/node_modules/@histoire/shared/dist/types/index.js',
    '/home/smares/my-app/node_modules/@histoire/shared/dist/state.js',
    '/home/smares/my-app/node_modules/@histoire/shared/dist/story.js',
    '/home/smares/my-app/node_modules/@histoire/shared/dist/type-utils.js',
    '/__resolved__virtual:$histoire-config',
    '/__resolved__virtual:$histoire-support-plugins-client',
    '/home/smares/my-app/node_modules/@vue/runtime-dom/package.json',
    '/home/smares/my-app/node_modules/@vue/runtime-dom/dist/runtime-dom.esm-bundler.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/HomeView.vue2.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/story/StoryView.vue2.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/base/BaseSplitPane.vue2.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/components/base/BaseSplitPane.vue3.js',
    '/home/smares/my-app/node_modules/@histoire/app/dist/bundled/_virtual/_plugin-vue_export-helper.js'
  ]
}

Node.js v18.14.2
```

### Reproduction

https://stackblitz.com/edit/github-nqxa3w?file=package.json

### System Info

```shell
System:
    OS: Linux 5.15 Ubuntu 22.10 22.10 (Kinetic Kudu)
    CPU: (20) x64 12th Gen Intel(R) Core(TM) i7-12700T
    Memory: 26.17 GB / 31.19 GB
    Container: Yes
    Shell: 5.2.2 - /bin/bash
  Binaries:
    Node: 18.14.2 - /usr/bin/node
    Yarn: 3.3.0 - /usr/bin/yarn
    npm: 9.5.0 - /usr/bin/npm
  Browsers:
    Chrome: 110.0.5481.177
  npmPackages:
    @histoire/plugin-nuxt: ^0.15.9 => 0.15.9 
    @histoire/plugin-vue: ^0.15.8 => 0.15.8 
    histoire: ^0.15.9 => 0.15.9
```


### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/histoire-dev/histoire/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/histoire-dev/histoire/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://histoire.dev/guide/).
- [X] Check that there isn't [already an issue](https://github.com/histoire-dev/histoire/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/histoire-dev/histoire/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by AnjaRoseBikes** [comment]
Same Bug with npm 

```
System:
    OS: Mac OS Version 13.2.1
  Binaries:
    Node: 18.15.0 - /usr/bin/node
    npm: 9.5.1 - /usr/bin/npm
  npmPackages:
    @histoire/plugin-nuxt: ^0.15.8 => 0.15.8 
    @histoire/plugin-vue: ^0.15.8 => 0.15.8 
    histoire: ^0.15.8 => 0.15.8
```

**Comment by Javid-Izadfar** [comment]
Same with pnpm
```
  System:
    OS: Linux 5.10 Ubuntu 20.04 LTS (Focal Fossa)
    CPU: (12) x64 Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz
    Memory: 2.22 GB / 7.72 GB
    Container: Yes
    Shell: 5.0.16 - /bin/bash
  Binaries:
    Node: 17.5.0 - ~/.nvm/versions/node/v17.5.0/bin/node
    Yarn: 1.22.17 - ~/.nvm/versions/node/v17.5.0/bin/yarn
    npm: 8.4.1 - ~/.nvm/versions/node/v17.5.0/bin/npm
  npmPackages:
    @histoire/plugin-nuxt: 0.15.9 => 0.15.9 
    @histoire/plugin-vue: 0.15.8 => 0.15.8 
    histoire: 0.15.9 => 0.15.9 
    
    ```

**Comment by mrkswrnr** [comment]
It looks like the vite define property `__NUXT_VERSION__` breaks `histoire build`. The property is defined by [nuxt](https://github.com/nuxt/nuxt/blob/2ebf5f4c4488c911e8bcc273cb8b12b86bf3562e/packages/vite/src/vite.ts#L64) since v3.3.0. My current workaround is to remove this property with a simple plugin:

```typescript
// histoire.config.ts
import { defineConfig } from 'histoire'
import { HstVue } from '@histoire/plugin-vue'
import { HstNuxt } from '@histoire/plugin-nuxt'

export default defineConfig({
  plugins: [
    HstVue(),
    HstNuxt(),
    {
      name: 'histoire-patch',
      config(config) {
        if (typeof config.vite === 'object' && config?.vite?.define)
          config.vite.define.__NUXT_VERSION__ = ''
      },
    },
  ],
})
```

**Comment by Akryum** [comment]
Hello, could you all try again with 0.16.1? :pray:

**Comment by AnjaRoseBikes** [comment]
Fixed! Thank you! ❤️

**Comment by SuperSchek** [comment]
I am still having this issue with `0.16.1`.

@mrkswrnr's workaround still works for me though.

**Comment by mrkswrnr** [comment]
I am still having this issue, too. 0.16.1.
`__NUXT_VERSION__` (which contains double quotes, e.g. `"3.4.1"`) still seems to break the build.

**Comment by kost1k** [comment]
I am still having this issue, too. `0.16.1`. 
Tested on Nuxt versions `3.4.1` , `3.4.2` , `3.4.3`.

@mrkswrnr's workaround is working

**Comment by medfreeman** [comment]
It seems it has been fixed, probably in [`releases/v0.17.2`](https://github.com/histoire-dev/histoire/releases/tag/v0.17.2), specifically in #588.

**Comment by Akryum** [comment]
Please comment/reopen if it's still an issue
